CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This CHANGELOG follows the format listed at [Our CHANGELOG Guidelines](https://g
 Which is based on [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- check-jenkins-build-time.rb: add `--check-build-duration` mode to check duration (@CoRfr)
```

Comment:
These should go under an `### Added` sub header per our [changelog guidelines](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)